Include/pylifecycle.h

```diff
@@ -81,7 +81,7 @@ PyAPI_FUNC(void) Py_EndInterpreter(PyThreadState *);
  * exit functions.
  */
 #ifndef Py_LIMITED_API
-PyAPI_FUNC(void) _Py_PyAtExit(void (*func)(void));
+PyAPI_FUNC(void) _Py_PyAtExit(PyObject *, void (*func)(PyObject *));
```

Comment:
A nit, but it would seem to me more conventional to take the user data (the `PyObject *`) as second argument.